src/Symfony/Component/Notifier/Bridge/Gitter/README.md

```diff
@@ -0,0 +1,23 @@
+Gitter Notifier
+========================
+
+Provides [Gitter](https://gitter.im/) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+GITTER_DSN=gitter://TOKEN@default?room=ROOM_ID
```

Comment:
What kind of "id" is it? An integer, hash or a string? I am wondering if `room_id` would be better in this case 🤔

Reply:
It is an hash. I switched to `room_id` and use a test hash instead of the string `testRoom` for the unit tests :v:

Reply:
I can provide a test room id if you don't want to start your own...

Reply:
I can provide a test `room_id` if you don't want to start your own...